Integrations/Flashpoint/Flashpoint.py

```diff
@@ -0,0 +1,1321 @@
+        }
+
+    else:
+        hr += 'No related reports found for the search.'
+        ec = {}
+
+    return hr, ec, resp
+
+
+def get_event_by_id_command(client, event_id):
+    """
+    Get specific event using its event id
+
+    :param client:
+    :param event_id: event's fpid
+    :return: command output
+    """
+    url_suffix = '/indicators/event/' + urllib.parse.quote(event_id.encode('utf-8'))
+    resp = client.http_request("GET", url_suffix=url_suffix)
+
+    hr = '### Flashpoint Event details\n'
+    ec: Dict[Any, Any] = {}
+
+    if len(resp) <= 0:
+        hr += 'No event found for the given ID.'
+        return hr, ec, resp
+
+    event = resp[0].get('Event', '')
+    fpid = resp[0].get('fpid', '')
+
+    events = []
+    if event:
+        event = convert_event(client, event, fpid)
+        events.append(event)
+        hr += tableToMarkdown('Below are the detail found:', events, ['Observed time (UTC)', 'Name', 'Tags'])
+
+        ec['Flashpoint'] = {
+            'event': {
+                'event_id': events[0]['event_id'],
+                'tag': events[0]['Tags'],
+                'date': events[0]['Observed time (UTC)'],
+                'event_creator_email': event['event_creator_email'],
+                'href': resp[0]['href']
+            }
+        }
+
+    return hr, ec, resp
+
+
+def get_events_command(client, limit, report_fpid, attack_ids, time_period):
+    """
+    Get events matching the given parameters
+
+    :param client:
+    :param limit: limit of the records
+    :param report_fpid:
+    :param attack_ids: array of attack ids of event
+    :param time_period:
+    :return: command output
+    """
+    url_suffix = '/indicators/event?sort_timestamp=desc&'
+    getvars = {}
+    if limit:
+        getvars['limit'] = limit
+
+    if report_fpid:
+        getvars['report'] = report_fpid
+
+    if attack_ids:
+        getvars['attack_ids'] = attack_ids
+
+    if time_period:
+        getvars['time_period'] = time_period
+
+    url_suffix = url_suffix + urllib.parse.urlencode(getvars)
+
+    resp = client.http_request("GET", url_suffix=url_suffix)
+    indicators = resp
+    hr = ''
+    if len(indicators) > 0:
+        hr += '### Flashpoint Events\n\n'
+
+        events = []
+        hrefs = []
+        for indicator in indicators:
+            hrefs.append(indicator.get('href', ''))
+            event = indicator.get('Event', {})
+            fpid = indicator.get('fpid', '')
+            event = convert_event(client, event, fpid)
+            events.append(event)
+
+        hr += tableToMarkdown('Below are the detail found:', events, ['Observed time (UTC)', 'Name', 'Tags'])
+
+        fp_link = client.url + '/home/search/iocs'
+        if attack_ids:
+            fp_link = fp_link + '?attack_ids=' + urllib.parse.quote(attack_ids)
+        hr += '\nAll events and details (fp-tools): [{}]({})\n'.format(fp_link, fp_link)
+        ec: Dict[Any, Any] = {'Flashpoint': {
+            'events': events
```

Comment:
events returned in this command and in `flashpoint-get-single-event` is the same event. That is why all those events should be under the same path.
`Flashpoint.event(val.event_id == obj.event_id)`. 
this way when user will call `flashpoint-get-events` and `flashpoint-get-single-event` they both output events to `Flashpoint.event` in context

Reply:
Changed the context-paths